modules/tinymce/src/core/test/ts/browser/delete/CefDeleteTest.ts

```diff
@@ -140,4 +141,84 @@ describe('browser.tinymce.core.delete.CefDeleteTest', () => {
       })
     );
   });
+
+  context('cef block is at the start/end of the content and covered with the selection', () => {
+    Arr.each([ Keys.delete, Keys.backspace ], (ks) => {
```

Comment:
This is going to be problematic when debugging, as you're not going to know where it's a delete or backspace that failed. As such, we probably need to turn this into an array of objects like the following and then replace `delete` in each `it` block with the `label`.
```
[
  { label: 'Delete', key: Keys.delete },
  { label: 'Backspace', key: Keys.backspace }
]
```

- [ ] Ensure the tests have some way to report/log what key is actually pressed